apps/condo/domains/meter/schema/MeterReadingFilterTemplate.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createschema meter.MeterReadingFilterTemplate 'name:Text; employee:Relationship:OrganizationEmployee:CASCADE; filters:Json'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/meter/access/MeterReadingFilterTemplate')
+const { METER_READING_FILTERS_FIELD } = require('./fields/MeterReadingFilters')
+
+
+const MeterReadingFilterTemplate = new GQLListSchema('MeterReadingFilterTemplate', {
```

Comment:
You named it …FilterTemplate. Therefore one record in DB will be a template for one filter (singular).
Look at a comment for `filters` field below.